### ISSUE TITLE
WinRT Exception

### ISSUE THREAD

**Issue by raver99** [colbylwilliams/VersionTrackingPlugin#5]
Hi,

For WinRT apps I get an exception while trying to store the versionTrail list into the app settings.
I get: "Error trying to serialize the value to be written to the application data store".

I think storing lists in the data store is not supported, so a serialization is needed.
I will add a PR with a fix.

Cheers,
Roland

**Comment by johnshardman** [comment]
Has this fix made it to a new NuGet package? If so, could you point me at it please.

**Comment by jerone** [comment]
I'm getting the same issue when using this plugin in UWP...

```
Exception thrown: 'System.Runtime.InteropServices.COMException' in System.Private.CoreLib.ni.dll
WinRT information: Error trying to serialize the value to be written to the application data store
An exception of type 'System.Runtime.InteropServices.COMException' occurred in System.Private.CoreLib.ni.dll but was not handled in user code
WinRT information: Error trying to serialize the value to be written to the application data store
Data of this type is not supported.

Error trying to serialize the value to be written to the application data store
```

It appears that the similar issue https://github.com/colbylwilliams/VersionTrackingPlugin/issues/8 was fixed by PR https://github.com/colbylwilliams/VersionTrackingPlugin/pull/9, which is closed without reason. 
Also this PR https://github.com/raver999/VersionTrackingPlugin/pull/1 from @raver999 looks interesting.

It works fine in iOS and Android.

--------

_Xamarin Forms 2.4.0.38779_
_Plugin.VersionTracking 2.0.1_

**Comment by jamesmontemagno** [comment]
I'll take a look. Is this only when tracking?

**Comment by jerone** [comment]
[**@jamesmontemagno**](https://github.com/jamesmontemagno) commented on [Nov 3, 2017, 2:48 PM GMT+1](https://github.com/colbylwilliams/VersionTrackingPlugin/issues/5#issuecomment-341708114 "2017-11-03T13:48:56Z - Replied by Github Reply Comments"):
> I'll take a look. Is this only when tracking?

This is immediately on the call `CrossVersionTracking.Current.Track();`.

**Comment by jamesmontemagno** [comment]
All fixed up :)

**Comment by jerone** [comment]
I can confirm it being fixed in v2.1.0. Thanks for fixing!
